frontend/src/components/SideBar/AddGroupTags.tsx

```diff
@@ -1,14 +1,19 @@
-import React from 'react';
+import React, { ReactElement } from 'react';
 import { Tag } from 'common/types/store-types';
 import { store } from '../../store/store';
 import styles from './AddGroupTags.module.scss';
+import { createGroup } from '../../firebase/actions';
 
 type AddGroupTagsProps = {
   readonly show: boolean;
   readonly setShow: (show: boolean) => void;
 };
 
-export default function AddGroupTags({ show, setShow }: AddGroupTagsProps): React.ReactElement {
+function newGroup(classCode: string): void {
+  createGroup('New group', new Date(), classCode);
```

Comment:
Maybe we should use classCode as group name?

Reply:
I thought we would let them specify their own group name?

Reply:
I think Sam means to use the class code as the default name. What do you guys think about `'My CS 3110 Group'` or `New CS 3110 Group` (if it's for CS 3110)

Reply:
[went with `New CS 3110 Group`](https://github.com/cornell-dti/samwise/pull/571/commits/bc9d22eadd457f85e3ee9ad7fbe98d6caef40264)